pages/common/gallery-dl.md

```diff
@@ -7,6 +7,10 @@
 
 `gallery-dl "{{url}}"`
 
+- Save images to a specific directory/{{relevant_folder}}, use --directory to avoid that:
```

Comment:
```suggestion
- Save images to a specific directory:
```

Not sure what you meant by the last half-sentence. Also, I removed the placeholder in the description.